fpmsyncd/routesync.cpp

```diff
@@ -829,7 +833,7 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
         fvVector.push_back(wt);
     }
 
-    if (!warmRestartInProgress)
+    if (!warmRestartInProgress  || (fastRestartInProgress && isDefaultRoute(destipprefix)))
```

Comment:
@arfeigin what happens to no default routes? Will they even be installed?

Reply:
Non default routes will be handled with existing logic (as prior to this change), by the else statement on line #847 they will be stored in a helper map and then set to route-table after timer will expire/End-of-Rib arrives.